frontend/src/storybook/avatar/avatar.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import {
+    Avatar,
+    StoreProvider,
+} from '~/bundles/common/components/components.js';
+import { Size } from '~/bundles/common/enums/size.enum.js';
```

Comment:
I think that you need to merge the development branch, because we don't have the Size enum it was replaced by ComponentSize